wall_e/src/resources/cogs/ban.py

```diff
@@ -101,13 +101,15 @@ async def intercept(self, guild: discord.Guild, member: Union[discord.User, disc
         # need to read the audit log to grab mod, date, and reason
         logger.info(f"[Ban intercept()] guild ban detected and intercepted for user='{member}'")
         try:
-            audit_ban = await self.bot.guilds[0].audit_logs(action=discord.AuditLogAction.ban).flatten()
+            date = datetime.datetime.now() - datetime.timedelta(minutes=1)
+            audit_ban = await guild.audit_logs(action=discord.AuditLogAction.ban, after=date,
+                                               oldest_first=False).flatten()
```

Comment:
so now that you have `oldest_first=False`, its now in descending order?

Reply:
Yea. I had to specify false because it defaults to true when you use the `after` param. 

This stuff only really comes into play if theres multiple bans happening in quick succession due to troll joins or bots, happened once last year.